Examples/Io/Root/include/ActsExamples/Io/Root/RootMeasurementWriter.hpp

```diff
@@ -72,16 +75,24 @@ class RootDigitizationWriter final : public WriterT<MeasurementContainer> {
     int surfaceID;
 
     /// Type 0 - free, 1 - bound
-    int measType = 1.;
+    int measType = 1;
 
-    // Truth parameters
+    /// Truth parameters
     float trueBound[Acts::eBoundSize];
     float trueGx = 0.;
     float trueGy = 0.;
     float trueGz = 0.;
 
+    /// Reconstruction information
     float recBound[Acts::eBoundSize];
 
+    // Cluster information
```

Comment:
A bit of description of the variables might be very helpful.

Reply:
Added.